test/software/amazon/ion/BadIonTest.java

```diff
@@ -75,7 +77,15 @@ public void testLoadString()
             }
             catch (IonException e)
             {
-                assert myTestFile.getPath().contains("bad/utf8") || myTestFile.getPath().contains("bad\\utf8");
+                // checks that test failed because of bad UTF-8 data
+                final CharBuffer buffer = CharBuffer.allocate(1204);
```

Comment:
What's this 1204?

Reply:
initial buffer size

Reply:
What I meant to say was *why* is this 1204?

Reply:
noticed the typo before merging, should be `1024` or `4kb`. 

It was arbitrarily chosen, doesn't look like it affected the tests when running locally.